src/interface/api/Output.cpp

```diff
@@ -101,16 +101,6 @@ void Output::setTimeOutTop(bool top) { timeOutAtTop = top; }
 
 bool Output::isTimeOutAtTop() { return timeOutAtTop; }
 
-void Output::setKeeperTree(std::string name) { Output::interfaceGameState.keeperStrategyName = std::move(name); }
-
-void Output::setStrategyTree(std::string name) {
```

Comment:
Did you also remove these from the .h file?

Reply:
now I have :)